### PR TITLE
Create fork remote template

### LINKED ISSUE #43: Feat: Create FORK_CLONE_REMOTE.md for all projects repo to refer to
### Is your feature request related to a problem? Please describe.
Currently projects under anita-org has their own copy of `Fork, Clone & Remote` instructions on wiki page. These pages are redundant because we only need one page that all projects can refer to.
 
### Describe the solution you'd like
Create FORK_CLONE_REMOTE.md template.

### Describe alternatives you've considered
NA

### Additional context
Make sure you remove any project specific references.

Screen Shot 2021-03-19 at 9 25 49 am

### PR DESCRIPTION
**Pull request by Jasmine9003**
### Description

Fixes #43

### Type of Change:

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**

- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

**Code/Quality Assurance Only**

- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Review by isabelcosta — APPROVED** [review]
Looks goo @Jasmine9003 🤗 
Thank you so much for contributing!

File: FORK_CLONE_REMOTE.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Remote
+
+<b> Note:</b> <i>This is only needed if you want to contribute to the project.</i>
+
+When a repository is cloned, it has a default remote named ```origin``` that points to your fork on GitHub, not the original repository it was forked from. To keep track of the original repository, you should add another remote named upstream. For this project it can be done by running the following command -
+
+```git remote add upstream https://github.com/anitab-org/<Project_Name>```
+
+You can check that the previous command worked by running ==git remote -v==. You should see the following output:
```
Comment (mtreacy002) [inline FORK_CLONE_REMOTE.md:43]: @Jasmine9003, IMO the use of `==` before and after the `==git remote -v==` could raise confusion. Perhaps we can use the command with backquote `git remote -v` instead?

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@Jasmine9003 , you're doing great on your first contribution 👍. You're almost there, can you please address my suggestion below? Thanks beforehand 😉.

File: FORK_CLONE_REMOTE.md (hunk 1 above)
Comment (Jasmine9003) [reply FORK_CLONE_REMOTE.md]: @mtreacy002 made the changes, please review.

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@Jasmine9003 , I just noticed one more thing. It seems that you've used only triple backquotes in this file. Normally the triple backquote is used only for quoting multiple lines (e.g. line 46 to 50). For a single word/phrase/line, you only need to use a single backquote (\` \`). So, can you please all the triple backquotes you used to a single backquote (except for the line 46 to 50. Thanks 😉.

**Comment by Jasmine9003** [comment]
@mtreacy002 made the suggested changes to change ``` to ` backquote.

**Comment by isabelcosta** [comment]
@mtreacy002 could you please review this again?

**Review by mtreacy002 — APPROVED** [review]
Thanks @Jasmine9003 for accommodating to the change requested. It looks good to me now. Thanks again for your contribution to the project 🎉🎉

**Comment by isabelcosta** [comment]
@Jasmine9003 awesome contribution! thank you for working on this 🙏🏾

**Pull request merged**